### ISSUE TITLE
Support null safety

### ISSUE THREAD

**Issue by md84419** [ookami-kb/flutter_arc_text#11]
```
The library 'package:flutter_arc_text/flutter_arc_text.dart' is legacy, and should not be imported into a null safe library.
Try migrating the imported library.
```

```
Error: This project cannot run with sound null safety, because one or more project dependencies do not
support null safety:

- flutter_arc_text
```

```
dependencies:
  flutter_arc_text: ^0.3.2
```

**Comment by ookami-kb** [comment]
Resolved in `0.4.0-nullsafety.0` version.

**Comment by md84419** [comment]
Thank you!
